### PR TITLE
Fix JS Cup

### PR DESCRIPTION
**Pull request by ktksan**
Change water to targetable.

### CODE REVIEW

**Comment by ktksan** [comment]
This has been done to support the PR: https://github.com/Terasology/EdibleSubstance/pull/1

File: assets/blocks/liquid/Water.block [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "hardness": 0,
     "attachmentAllowed": false,
     "translucent": true,
-    "targetable": false,
+    "targetable": true,
```
Comment (skaldarnar) [inline assets/blocks/liquid/Water.block:10]: Unfortunately, this leads to water blocks not being replaceable anymore 😢

**Review by skaldarnar — CHANGES_REQUESTED** [review]
Just tested this out together with https://github.com/Terasology/EdibleSubstance/pull/1, and while drinking water with the cup works fine, water blocks cannot be replaced anymore. Neither when standing on shore and looking into water, nor when underwater. Also, it's not possible to mine blocks when underwater (the targeted block is always the water block).

We'll need to find another way to make this happen... I'm wondering whether we can reverse engineer what @Josharias did implement there...

**Comment by jdrueckert** [comment]
Fixed the root cause (filling containers) in
https://github.com/Terasology/Fluid/pull/15 and
https://github.com/Terasology/ManualLabor/pull/27
which resulted in drinking from WoodenCup being possible again as a side effect.

Anyway, thank you for trying to fix it this way :) I'll close this PR as this is no longer an issue.

**Pull request closed without merging**